facebook-commerce-pixel-event.php

```diff
@@ -657,12 +657,14 @@ private static function get_version_info() {
 		 * Get PixelID related settings.
 		 */
 		public static function get_options() {
-			return get_option( self::SETTINGS_KEY ) ?: array(
-				self::PIXEL_ID_KEY     => '0',
-				self::USE_PII_KEY      => 0,
-				self::USE_S2S_KEY      => false,
-				self::ACCESS_TOKEN_KEY => '',
-			);
+			return is_array( get_option( self::SETTINGS_KEY ) ) && ! empty( get_option( self::SETTINGS_KEY ) )
+				? get_option( self::SETTINGS_KEY )
+				: array(
+					self::PIXEL_ID_KEY     => '0',
+					self::USE_PII_KEY      => 0,
+					self::USE_S2S_KEY      => false,
+					self::ACCESS_TOKEN_KEY => '',
+				);
```

Comment:
```suggestion
            $settings = get_option( self::SETTINGS_KEY );
			return is_array( $settings ) && ! empty( $settings )
				? $settings
				: array(
					self::PIXEL_ID_KEY     => '0',
					self::USE_PII_KEY      => 0,
					self::USE_S2S_KEY      => false,
					self::ACCESS_TOKEN_KEY => '',
				);
```

Reply:
Thanks @ibndawood. I address this in 1f3db20f04cac5e7010386ef693dadab83131741